svir/dialogs/drive_oq_engine_server_dialog.py

```diff
@@ -135,6 +137,22 @@ def attempt_login(self):
         else:
             self.refresh_calc_list()
 
+    def check_engine_compatibility(self):
+        engine_version = self.get_engine_version()
+        assert engine_version is not None
+        engine_major, engine_minor, _ = engine_version.split('.')
+        engine_major, engine_minor = int(engine_major), int(engine_minor)
```

Comment:
The recommended way of doing version checks is to use tuples:

>>> tuple(int(x) for x in '2.5.0'.split('.'))
(2, 5, 0)

Tuples in Python are ordered, and you can also use `==` among them.